x/wasm/keeper/msg_server_test.go

```diff
@@ -57,3 +61,80 @@ func TestSelectAuthorizationPolicy(t *testing.T) {
 		})
 	}
 }
+
+func TestSetGaslessAndUnsetGasLessProposal(t *testing.T) {
```

Comment:
Let's add an unauthorized test case

Reply:
I'v added an unthorized test case. Could you check it pls